docs/current/overview.md

```diff
@@ -1 +1,66 @@
+(bolded) on each computer.
+
+On the remote server, **global pathfinding** uses the A* pathfinding algorithm to create a sailing path, a list of global
+waypoints from the current position to destination. Global sailing paths are sent via the **Remote Transceiver** to the
+**Local Transceiver** on the on board computer.
+
+On the on board computer, the **CAN Transceiver** receives GPS and wind data from their respective sensors. This raw data
+is filtered before being used in the other software components. **Local Pathfinding** uses GPS and wind data, as well as
+the global path and AIS data from the **AIS Receiver**, to create a local path, a list of local waypoints from the current
+position to the next global waypoint. The **Controller** uses wind data and the relative bearing to the local path to adjust
+the rudder and sails accordingly. The state of the boat and research data we collect is sent via the **Local Transceiver**
+to the **Remote Transceiver** on the remote server.
+
+Back on the remote server, the **Website** presents the boat state and research data for monitoring and analysis purposes.
+The **Remote Transceiver** additionally includes manual overrides such as resetting the boat state and modifying the global
+path.
+
+As for the communication mediums, the computers communicate via satellite, and components on the on board computer communicate
+through the [Robot Operating System framework](../reference/ros.md){target=_blank}, or ROS for short.
+
+For software development purposes, all software components will be able to run and communicate with each other locally.
+To accomplish this, we will:
+
+1. Create a development environment that has all software component dependencies: **Sailbot Workspace**.
+2. Develop accurate simulations of the environment and hardware: **Simulator**, **Mock AIS**, **Mock Global Pathfinding**.
+3. Add configuration options to select between real and simulated hardware as well as running remote server components
+   remotely or locally.
+
+## Diagrams
+
+In these diagrams, the bubbles represent components of our software system, and the direction of arrows connecting the
+bubbles represent the flow of data between them. The color of the bubbles denote the sub-team leading their development:
+
+- Purple: Controls
+- Green: Network Systems
+- Blue: Pathfinding
+- White: Outside the Software Team's scope
+
+Components that are used in both the production and development environments are darker,
+while ones that are only used in one are lighter.
+
+!!! tip "Interacting with the diagram"
+
+    - To switch between the production and development environment diagrams, hover over the image and navigate with
+      the arrows in the top left
+    - Expand the diagram by clicking on it
+
+<!-- This was generated by going to the source, selecting File > Embed > HTML, and deselecting Layers, Tags, and Edit -->
```

Comment:
There is an option to embed the public URL of the diagram, but I wasn't able to get that working. It didn't preview, and when I pasted it into the website I got this error

```
Failed to execute 'serializeToString' on 'XMLSerializer': parameter 1 is not of type 'Node'.
```

Reply:
@DFriend01 can you try to get the public URL working? If it doesn't seem possible no worries

Reply:
Seems like you got this working. Can you confirm?

Reply:
I replaced the HTML with an iframe. The link was much more condensed.

Reply:
Ooh this is exactly what I wanted, thanks